src/ui/Box/Box.tsx

```diff
@@ -1,5 +1,36 @@
 import { styled } from '../../stitches.config';
 
-export const Box = styled('div', {});
+export const Box = styled('div', {
+  variants: {
+    input: {
+      true: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        width: '100%',
+      },
+    },
+    errors: {
+      true: {
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        margin: 0,
+        minHeight: 45,
```

Comment:
not sure what this accomplishes

Reply:
it's meant to center then input and the label, keep them in a column..

Reply:
i mean minHeight specifically. were you trying to prevent the form from resizing when errors appear/disappear?

Reply:
I tried to replicate the old styling.But you are right, this should be removed because it make the error text and textarea above when resizing the page to smaller heights